src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/DefaultRazorProjectServiceTest.cs

```diff
@@ -29,12 +29,11 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer;
 
 public class DefaultRazorProjectServiceTest : LanguageServerTestBase
 {
-    private readonly IReadOnlyList<DocumentSnapshotHandle> _emptyDocuments;
+    private readonly ImmutableArray<DocumentSnapshotHandle> _emptyDocuments = ImmutableArray<DocumentSnapshotHandle>.Empty;
```

Comment:
Consider just inlining `ImmutableArray<DocumentSnapshotHandle>.Empty`. Eventually, when we can update System.Collections.Immutable to 8.0, we'll replace all of them with `[]` in one go.